drivers/usb_c/tcpc/shell.c

```diff
@@ -14,6 +14,25 @@
 /** Macro used to call the dump_std_reg function from the USB-C connector node */
 #define TCPC_DUMP_CONN_NODE(node) TCPC_DUMP_DEV(DEVICE_DT_GET(DT_PROP(node, tcpc)))
 
+/** Macro used to call the vbus_measure function from the VBUS device pointer */
+#define TCPC_VBUS_DEV(dev)                                                                         \
+	{                                                                                          \
+		int val;                                                                           \
+		ret |= usbc_vbus_measure(dev, &val);                                               \
+		shell_print(sh, "%s vbus: %d mV", dev->name, val);                                 \
+	}
+
+/** Macro used to call the vbus_measure function from the USB-C connector node */
+#define TCPC_VBUS_CONN_NODE(node) TCPC_VBUS_DEV(DEVICE_DT_GET(DT_PROP(node, vbus)))
+{
+	ret |= tcpc_get_chip_info(DEVICE_DT_GET(DT_PROP(node, tcpc)), &chip_info);
+	shell_print(sh, "Chip: %s", DEVICE_DT_GET(node)->name);
+	shell_print(sh, "\tVendor:   %04x", chip_info.vendor_id);
+	shell_print(sh, "\tProduct:  %04x", chip_info.product_id);
+	shell_print(sh, "\tDevice:   %04x", chip_info.device_id);
+	shell_print(sh, "\tFirmware: %llx", chip_info.fw_version_number);
```

Comment:
This shouldn't be in this commit.  This is TCPC chip info and belongs in the next commit.